langchain/model_laboratory.py

```diff
@@ -13,16 +13,18 @@
 class ModelLaboratory:
     """Experiment with different models."""
 
-    def __init__(self, chains: Sequence[Chain], names: Optional[List[str]] = None):
+    def __init__(
+        self, chains: Sequence[MultiVariableChain], names: Optional[List[str]] = None
+    ):
         """Initialize with chains to experiment with.
 
         Args:
             chains: list of chains to experiment with.
         """
         for chain in chains:
-            if not isinstance(chain, Chain):
+            if not isinstance(chain, MultiVariableChain):
```

Comment:
why do these have to be multivariable?

Reply:
Bc you have the option of working with LLMChains, which are multivariable rn

Reply:
but would this code not work with SingleVariableChain?